src/main/java/ejercicio/banco/repository/MySqlBankRepository.java

```diff
@@ -48,13 +48,12 @@ public Bank find(int id) {
         try {
             PreparedStatement pstmt = con.prepareStatement("select * from BANKS where id = " + id);
             ResultSet rs = pstmt.executeQuery();
-            if (rs.next()) {
+            //if (rs.next()) {
```

Comment:
Be careful with commented code :)

Reply:
This was when I was still trying to understand how result sets worked, should be fixed in present state